packages/teleport/src/services/recordings/makeRecording.ts

```diff
@@ -80,7 +80,7 @@ function makeSshRecording({
     users: participants.join(', '),
     hostname,
     description,
-    recordingType: 'ssh',
+    recordingType: kubernetes_cluster ? 'k8s' : 'ssh',
```

Comment:
We should rename this function to something more accurate like `makeSshOrKubeRecording`, or abstract out the common logic and call it from `makeSshRecording` and a new `makeKubeRecording`

Reply:
renamed to `makeSshOrKubeRecording`